acl_loader/main.py

```diff
@@ -593,6 +609,14 @@ def convert_rule_to_db_schema(self, table_name, rule):
 
         rule_props["PRIORITY"] = str(self.max_priority - rule_idx)
 
+        # add ip type match to dataplane acl if not specified in the rule
+        if self.is_table_l3v6(table_name):
+            if "IP_TYPE" not in rule_props:
+                rule_props["IP_TYPE"] = "IPV6ANY"  # ETHERTYPE is not supported for DATAACLV6
+        elif self.is_table_l3(table_name):
+            if "ETHER_TYPE" not in rule_props:
+                rule_props["ETHER_TYPE"] = str(self.ethertype_map["ETHERTYPE_IPV4"])
```

Comment:
rule props won't have IP_TYPE or ETHER_TYPE until we do the `convert_action` and `deep_update` step below

Reply:
however *if* these exist below they will be overwritten. I still think it might be more clear to do this logic after we process all the openconfig fields